modules/process/pages/process-testing-index.adoc

```diff
@@ -18,6 +18,23 @@ If you have already created your own xref:ROOT:actors-index.adoc[actors] for you
 . Select the process pool
 . Click on the *Configure* icon of the Studio coolbar and configure the _Actor mapping_
 
+== Integration Testing
+
+[NOTE]
+====
+
+For Enterprise, Performance, Efficiency and Teamwork editions only.
+====
+
+To write automated integration test scenarios for your automation project you can use the Bonita Test Toolkit. It provides powerful methods to execute all the steps of a process and check the corresponding outputs (statuses, business data, tasks used)
+
+It can be used in any IDE and can easily be integrated into a Continuous Integration.
```

Comment:
DO not use easily in a technical documentation, you do not know the level of your reader. 
Especially when it's absolutely not easy, even for us ^^

```suggestion
It can be used in any IDE and can be integrated into a Continuous Integration.
```